src/LiteCore/src/LiteCore.Shared/Interop/Fleece.cs

```diff
@@ -397,6 +397,9 @@ public static void FLEncode(this object obj, FLEncoder* enc)
                 case DateTimeOffset dto:
                     (dto.ToString("o")).FLEncode(enc);
                     break;
+                case ReplicatorOptionProgressLevel level:
```

Comment:
Don't put Couchbase specific types here.  They can go in FLValueConverter.cs inside of `FLEncode`.  This is a dubious thing to have here though because it seems like it should be converted beforehand.  I don't want to add every enum into this list.

Reply:
removed.